### ISSUE TITLE
React does not recognize the `withRequiredValidator` prop on a DOM element.

### ISSUE THREAD

**Issue by kmarple1** [NewOldMax/react-material-ui-form-validator#76]
I'm getting this warning on a TextValidator:

```
<TextValidator
    fullWidth
    value={this.state.teamName}
    inputProps={{ maxLength: 40 }}
    label="Team Name"
    name="teamName"
    withRequiredValidator={true}
    validators={['required', 'matchRegexp:^[*0-9a-zA-Z_\\- ]+$']}
    errorMessages={['Required!', 'Enter up to 40 valid characters. Valid characters are A-Z, a-z, 0-9, _, -, or *.']}
    onChange={this.handleNameChange}
    error={this.state.error}
    helperText={this.state.helperText}
/>
```

"@material-ui/core": "1.3.0",
"react-material-ui-form-validator": "2.0.0-beta.10",

```
index.js:2178 Warning: React does not recognize the `withRequiredValidator` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `withrequiredvalidator` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    in div (created by FormControl)
    in FormControl (created by WithStyles(FormControl))
    in WithStyles(FormControl) (created by TextField)
    in TextField (created by TextValidator)
    in TextValidator (at teamDetail.jsx:222)
    in div (at teamDetail.jsx:221)
    in form (created by ValidatorForm)
    in ValidatorForm (at teamDetail.jsx:217)
    in div (at teamDetail.jsx:216)
    in div (at teamDetail.jsx:197)
    in TeamDetail (created by Connect(TeamDetail))
    in Connect(TeamDetail) (at teamList.jsx:173)
    in div (at teamList.jsx:172)
    in div (created by Paper)
    in Paper (created by WithStyles(Paper))
    in WithStyles(Paper) (created by Drawer)
    in Transition (created by Slide)
    in EventListener (created by Slide)
    in Slide (created by WithTheme(Slide))
    in WithTheme(Slide) (created by Drawer)
    in RootRef (created by Modal)
    in div (created by Modal)
    in Portal (created by Modal)
    in Modal (created by WithStyles(Modal))
    in WithStyles(Modal) (created by Drawer)
    in Drawer (created by WithStyles(Drawer))
    in WithStyles(Drawer) (at teamList.jsx:264)
    in div (at teamList.jsx:226)
    in TeamList (created by WithStyles(TeamList))
    in WithStyles(TeamList) (created by Connect(WithStyles(TeamList)))
    in Connect(WithStyles(TeamList)) (created by Route)
    in Route (at routes.jsx:21)
    in Switch (at routes.jsx:13)
    in div (at App.js:59)
    in div (at App.js:57)
    in Router (created by ConnectedRouter)
    in ConnectedRouter (at App.js:56)
    in IntlProvider (at App.js:55)
    in App (created by Connect(App))
    in Connect(App) (at index.js:22)
    in MuiThemeProvider (at index.js:21)
    in Provider (at index.js:20)
    in Root (created by Connect(Root))
    in Connect(Root) (at index.js:47)
```

UPDATE: This issue is that withRequiredValidator is being passed as a prop to material-ui's TextField component. I've opened a pull request to resolve the issue.

**Comment by NewOldMax** [comment]
closed by #77
